legion/tests/e2e/robot/tests/common/common_config.robot

```diff
@@ -0,0 +1,35 @@
+*** Variables ***
```

Comment:
Please fix problems with generation of swagger clients for Python (another place)